components/dashboard/SideNav.tsx

```diff
@@ -25,6 +27,13 @@ const NavSection = ({sectionName} : {sectionName: string}) => {
 }
 
 export default function SideNav() {
+    const [isInOrg, setIsInOrg] = useState(false);
+
+    useEffect(() => {
+        getUserIDandOrgID().then(({orgID}) => {setIsInOrg(orgID === null ? false : true)});
```

Comment:
Does this happen right after you create an account for the first time? If it is i think it happens cuz we redirect user to dashboard faster than supabase updates the database with the new user so technically user still doesnt exist by the time we redirect them

Reply:
Will be fixed when ND implements profile creation flow

Reply:
I actually think it's because I manually deleted my account from supabase and it isn't putting my account back in, I tried making a few more new accounts and can't produce the error.